src/Softeq.NetKit.Chat.Data.Persistent.Sql/Repositories/ChannelMemberRepository.cs

```diff
@@ -70,6 +70,24 @@ public async Task<ChannelMember> GetChannelMemberAsync(Guid memberId, Guid chann
             }
         }
 
+        public async Task<ChannelMember> GetChannelMemberWithMemberDetailsAsync(Guid memberId, Guid channelId)
+        {
+            using (var connection = _sqlConnectionFactory.CreateConnection())
+            {
+                var sqlQuery = $@"SELECT *
+                                 FROM ChannelMembers cm
```

Comment:
Let's select all fields clearly instead of *

Reply:
done